app/models/amusement_park.rb

```diff
@@ -1,3 +1,12 @@
-class AmusementPark < ApplicationRecord
+class AmusementPark< ApplicationRecord
   has_many :rides
+  has_many :ride_mechanics, through: :rides
```

Comment:
I think the association that you could add here that would essentially be the same is:
```ruby
has_many :mechanics, through: :rides 
```